meta/documents/user_guide/en/tips_and_tricks.md

```diff
@@ -0,0 +1,10 @@
+# Wayfair plugin: Tips and Tricks
+
+This document contains tips and ideas regarding use of the Wayfair plugin.
+
+## Protecting your credentials
+The credentials that your Plentymarkets system uses for communication with Wayfair should not be shared with anyone. You should take care to avoid doing these things:
+
+* Forwarding Wayfair emails

```

Comment:
Is this referring to particular emails from Wayfair?

Reply:
not really. I can try to rename this

Reply:
I'm making it say "Forwarding communications from Wayfair to others."